### PR TITLE
macbook air 7

### PR DESCRIPTION
**Pull request by theotheroracle**
###### Things done
added macbook air 7 and added to flake.nix and readme, also added readme in folder describing issues with wifi driver

- [ x ] Tested the changes in your own NixOS Configuration
- [ x ] Tested the changes end-to-end by using your fork of `nixos-hardware` and
      importing it via `<nixos-hardware>` or Flake input

### CODE REVIEW

File: apple/macbook-air/7/default.nix [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{ config, lib, pkgs, ... }:
+
+{
+  imports = [
+    ../.
+  ];
+
+  boot.extraModulePackages = lib.mkDefault [ config.boot.kernelPackages.broadcom_sta ];
```
Comment (Mic92) [inline apple/macbook-air/7/default.nix:8]: Does nixos-generate-config not also include this already?

Comment (theotheroracle) [reply apple/macbook-air/7/default.nix:8]: i'm not sure, the wifi didn't work initially, but i was able to get it working later, the macbook is currenly not booting, but if i get it fixed i'll let you know

Comment (Mic92) [reply apple/macbook-air/7/default.nix:8]: Afaik it should be in your hardware-configuration.nix. At least this is what I remember reading the perl code.

Comment (theotheroracle) [reply apple/macbook-air/7/default.nix]: ok, i removed it

**Comment by Mic92** [comment]
@mergify queue

**Pull request merged**